wire-ios-data-model/Source/E2EIdentity/GracePeriodRepository.swift

```diff
@@ -18,15 +18,16 @@
 
 import Foundation
 
-/// The repository is responsible for storing the grace period during which the user must enroll the end-to-end identity certificate.
-/// The grace period starts at the moment where the client fetches and processes the team feature flag.
+// sourcery: AutoMockable
 public protocol GracePeriodRepositoryInterface {
 
     func fetchGracePeriodEndDate() -> Date?
     func storeGracePeriodEndDate(_ date: Date)
 
 }
 
+/// The repository is responsible for storing the grace period during which the user must enroll the end-to-end identity certificate.
+/// The grace period starts at the moment where the client fetches and processes the team feature flag.
```

Comment:
Actually the documentation should be at the protocol. It describes what a conforming type should be capable of, independently of the actual implementation. 

If you are worried about the `sourcery` comment, it worked for me like that:
```
// sourcery: AutoMockable
/// Determines if a user has a valid E2EI certificate on all clients.
public protocol IsUserE2EICertifiedUseCaseProtocol {
```

Reply:
done